jicofo/rootfs/etc/cont-init.d/10-config

```diff
@@ -14,7 +16,7 @@ fi
 tpl /defaults/jicofo.conf > /config/jicofo.conf
 
 if [[ ! -f /config/logging.properties ]]; then
-    cp /defaults/logging.properties /config
+    tpl /defaults/logging.properties > /config/logging.properties
```

Comment:
Let's template the file unconditionally?

Reply:
I added a commit to remove the conditions.